cmd/adsysd/integration_tests/adsys_test.go

```diff
@@ -317,15 +317,21 @@ func changeAppArgs(t *testing.T, s setterArgs, conf string, args ...string) {
 }
 
 var (
-	systemSockets      = make(map[string]string)
-	systemAnswersModes = []string{
+	dbusSockets     = make(map[string]string)
+	dbusAnswerModes = []string{
 		"polkit_yes",
 		"polkit_no",
 		"no_startup_time",
 		"invalid_startup_time",
 		"no_nextrefresh_time",
 		"invalid_nextrefresh_time",
 		"subscription_disabled",
+
+		// This values will only influence the session bus
+		"no_vfs_bus",
```

Comment:
can we prefix them all with `gfvs_`?

(I think the others not prefixed will need to be prefix with `systemd` in a separate PR to make more sense too now that this is bigger)